src/assets/styles/styles/custom/_variables.scss

```diff
@@ -389,8 +386,8 @@ $box-shadow: 0 1px 20px 0px rgba(0, 0, 0, 0.1);
 $sidebar-box-shadow: 0px 2px 22px 0 rgba(0, 0, 0, 0.2), 0px 2px 30px 0 rgba(0, 0, 0, 0.35);
 
 $component-active-color: $white !default;
-$component-active-bg: theme-color("primary") !default;
```

Comment:
Musiałem pozbyć się części funkcji, ponieważ plik się nie kompilował

Reply:
Moim zdaniem, nie jest to duży problem, ponieważ i tak nie była używana